### PR TITLE
Update docs to mention kprobes and uprobes

### PR DESCRIPTION
**Pull request by anfredette**

### CODE REVIEW

File: docs/getting-started/bpfctl-guide.md [hunk 1]
```diff
@@ -86,27 +86,24 @@ There are a common set of attributes, and those MUST come before the program typ
 bpfctl load-from-file --help
 Load an eBPF program from a local .o file
 
-Usage: bpfctl load-from-file [OPTIONS] --path <PATH> --section-name <SECTION_NAME> <COMMAND>
+Usage: bpfctl load-from-file [OPTIONS] --path <PATH> --name <NAME> <COMMAND>
```
Comment (astoycos) [inline docs/getting-started/bpfctl-guide.md:89]:
```suggestion
Usage: bpfctl load-from-file [OPTIONS] --path <PATH> --name <PROGRAM_NAME> <COMMAND>
```

File: docs/getting-started/bpfctl-guide.md [hunk 2]
```diff
@@ -115,18 +112,24 @@ Options:
...
+          Optional: Program id of loaded eBPF program this eBPF program will share a map with.
+          Only used when multiple eBPF programs need to share a map.
+          Example: --map-owner-id 63178
 
   -h, --help
           Print help (see a summary with '-h')
 ```
 
-So when using `bpfctl load-from-file`, `--path`, `--section-name`, `--id`, `--global`
+When using `bpfctl load-from-file`, `--path`, `--section-name`, `--id`, `--global`
```
Comment (astoycos) [inline docs/getting-started/bpfctl-guide.md:132]:
```suggestion
When using `bpfctl load-from-file`, `--path`, `name`, `--id`, `--global`
```

File: docs/getting-started/bpfctl-guide.md (hunk 1 above)
Comment (anfredette) [reply docs/getting-started/bpfctl-guide.md:89]: I copied my update directly from the output of the command.  I just double checked, and what I have is what bpfctl says.

Comment (astoycos) [reply docs/getting-started/bpfctl-guide.md:89]: Ok that's alright.

We probably need to fix this to say `Program_name` in the documentation just to help avoid confusion.

**Review by astoycos — APPROVED** [review]
LGTM

File: docs/getting-started/bpfctl-guide.md (hunk 2 above)
Comment (anfredette) [reply docs/getting-started/bpfctl-guide.md:132]: We atually need to do a scrub of all the cli commands and output in the docs since things have changed.

**Pull request merged**